.devcontainer/default/devcontainer.json

```
@@ -0,0 +1,33 @@
+// Reference: https://containers.dev/implementors/json_reference/
+{
+  "name": "SourceKit-LSP",
+  "dockerFile": "Dockerfile",
+
+  // Allow the processes in the container to attach a debugger
+  "capAdd": [ "SYS_PTRACE" ],
+  "securityOpt": [ "seccomp=unconfined" ],
+
+  "mounts": [
+    // Use a named volume for the build products for optimal performance (https://code.visualstudio.com/remote/advancedcontainers/improve-performance?WT.mc_id=javascript-14373-yolasors#_use-a-targeted-named-volume)
+    "source=${localWorkspaceFolderBasename}-build,target=${containerWorkspaceFolder}/.build,type=volume",
+    // Do the same for experimental background indexing
+    "source=${localWorkspaceFolderBasename}-build,target=${containerWorkspaceFolder}/.index-build,type=volume"
```

Does this volume need to have a different name? At the moment it seems like `.build` and `.index-build` are mounted to the same volume, which means that they have the same contents.

```suggestion
    "source=${localWorkspaceFolderBasename}-index-build,target=${containerWorkspaceFolder}/.index-build,type=volume"
```